src/governance.ts

```diff
@@ -96,7 +108,13 @@ export class GovernanceCanister {
         canisterId,
       });
 
-    return new GovernanceCanister(canisterId, service, certifiedService);
+    return new GovernanceCanister(
+      canisterId,
+      service,
+      certifiedService,
+      agent,
+      options.hardwareWallet
```

Comment:
Probably it would be cleaner to set it to `false` by default

Reply:
It is default in the constructor.

If this is `undefined` it will be `false`.